lib/cretonne/meta/cdsl/instructions.py

```diff
@@ -203,11 +204,30 @@ def _verify_polymorphic(self):
                     raise RuntimeError(
                             "typevar_operand must be a free type variable")
             tv = self.outs[0].typevar
+            self.ctrl_typevar_ind = -1
             if tv is not tv.free_typevar():
                 raise RuntimeError("first result must be a free type variable")
             self.other_typevars = self._verify_ctrl_typevar(tv)
             self.ctrl_typevar = tv
 
+    def get_ctrl_typevar_ind(self):
```

Comment:
Renamed to get_ctrl_typevar_index and moved into tc.py; Added clarifying doc comment about indexing and now using value_opnums/value_results.